packages/components/src/aspect-ratio/stories/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Internal dependencies
+ */
+import { AspectRatio } from '../index';
+
+export default {
+	component: AspectRatio,
+	title: 'Components/AspectRatio',
```

Comment:
We've been namespacing components in Storybook under the Experimental heading.

```suggestion
	title: 'Components (Experimental)/AspectRatio',
```